.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - v*
+  workflow_dispatch:
+
+jobs:
+  publish_jvm:
+    name: Publish
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 10000
+      # Fetch all tags so that sbt-dynver can find the previous release version
+      - run: git fetch --tags -f
+      # Install OpenJDK 11
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: '11'
```

Comment:
Hi ~ @xerial,  1.19.0 was compiled and released using Java 11,  will Java 8 no longer be supported after 1.19.0?  I didn't find relevant information in the release notes.

Reply:
Sorry, please ignore my comment temporarily

Reply:
The build target is 1.8, so it should work with Java 8 https://github.com/xerial/snappy-java/blob/master/build.sbt#L30

Reply:
Thanks very much for your reply.

Reply:
@xerial I'm very sorry, but I still need to bother you
I wrote a simple demo to reproduce this issue, the demo is in https://github.com/LuciferYang/snappy-java-test

```
@Test
    public void testCompress() throws IOException {
        ByteBuffer src = ByteBuffer.allocateDirect(1024);
        src.put("hello world".getBytes());
        src.flip();
        ByteBuffer dest = ByteBuffer.allocateDirect(1024);
        int maxCompressedLen = Snappy.compress(src, dest);
        Assert.assertEquals(maxCompressedLen, 13);
    }
```

run the above case with zulu 8u352

```
java -version
openjdk version "1.8.0_352"
OpenJDK Runtime Environment (Zulu 8.66.0.15-CA-macos-aarch64) (build 1.8.0_352-b08)
OpenJDK 64-Bit Server VM (Zulu 8.66.0.15-CA-macos-aarch64) (build 25.352-b08, mixed mode)
```

with snappy-java 1.1.8.4 the test passed
```
mvn clean test -Dsnappy-java.version=1.1.8.4
```

with snappy-java 1.1.9.0
```
mvn clean test -Dsnappy-java.version=1.1.9.0
```
the test failed as follows:

```
-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Running org.example.SnappyJavaTest
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.374 sec <<< FAILURE!
testCompress(org.example.SnappyJavaTest)  Time elapsed: 0.345 sec  <<< ERROR!
java.lang.NoSuchMethodError: java.nio.ByteBuffer.limit(I)Ljava/nio/ByteBuffer;
	at org.xerial.snappy.Snappy.compress(Snappy.java:157)
	at org.example.SnappyJavaTest.testCompress(SnappyJavaTest.java:17)
```

Reply:
And if I run the mvn test with zulu 11, there is no above issue

Reply:
I create a issue https://github.com/xerial/snappy-java/issues/389